### PR TITLE
installation: fix dependencies

### PR DESCRIPTION
**Pull request by jma**
Signed-off-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -60,9 +60,9 @@
 ]
 
 install_requires = [
-    'celery<4.0,>=3.1',                 # FIXME: invenio-indexer
-    'elasticsearch<3.0.0,>=2.0.0',      # FIXME: invenio-search
-    'elasticsearch-dsl<3.0.0,>=2.0.0',  # FIXME: invenio-search
+    'celery>=3.1',
```
Comment (lnielsen) [inline setup.py]: Thanks for the PR. We've changed strategy, and now depend directly on Flask-CeleryExt where we can then easily control the Celery dependency, so this should be ``Flask-CeleryExt>=0.3.0``(previously for the problems you've had)....

File: setup.py [hunk 2]
```diff
@@ -60,9 +60,9 @@
...
+    'elasticsearch>=2.0.0,<6.0',
```
Comment (lnielsen) [inline setup.py]: Not sure what is the best way to proceed here. In Invenio-App-ILS (which is an instance) we do like this:
https://github.com/inveniosoftware/invenio-app-ils/blob/master/setup.py#L57-L65

Here, in a module, it's perhaps enough to just depend on Invenio-Search and remove the ES dependencies, unless the module explicitly doens't support ES5 (we have some of those)

File: setup.py (hunk 1 above)
Comment (jma) [reply setup.py]: Ok I will change it in that way.

File: setup.py (hunk 2 above)
Comment (jma) [reply setup.py]: Ok too. `invenio-search`dependency should be enough.

**Review by lnielsen — APPROVED** [review]

**Pull request merged**